src/openrct2/util/Util.cpp

```diff
@@ -223,18 +223,26 @@ bool sse41_available()
 bool avx2_available()
 {
 #ifdef OPENRCT2_X86
-// For GCC and similar use the builtin function, as cpuid changed its semantics in
-// https://github.com/gcc-mirror/gcc/commit/132fa33ce998df69a9f793d63785785f4b93e6f1
-// which causes it to ignore subleafs, but the new function is unavailable on Ubuntu's
-// prehistoric toolchains
+    // For GCC and similar use the builtin function, as cpuid changed its semantics in
+    // https://github.com/gcc-mirror/gcc/commit/132fa33ce998df69a9f793d63785785f4b93e6f1
+    // which causes it to ignore subleafs, but the new function is unavailable on Ubuntu's
```

Comment:
Time to change this unprofessional comment (if it even still applies).

Reply:
I have really no idea what this is about, I believe this was from @janisozaur

Reply:
Who else would write comments like this?

It would have been half-informative if it had mentioned the GCC version that was needed, but it seems sh*tting on Ubuntu was more important.

Reply:
Well shit happens, lets try to keep it civil. Should I ditch the comment?

Reply:
I think this still applies to Ubuntu 18.04 as well, so the comment should stay.

Personally, I'm not too bothered by the "prehistoric toolchain" comment, but feel free to replace that particular line with something more neutral.

Reply:
```diff
-but the new function is unavailable on Ubuntu's prehistoric toolchains
+but the new function is unavailable on Ubuntu 18.04's toolchains.
```